### ISSUE TITLE
[Feature Requeset] add tooltip for buttons on bottom toolbar

### ISSUE THREAD

**Issue by e-cloud** [zalmoxisus/redux-devtools-extension#355]
when the devtool panel width is a bit small, the label of those buttons are hidden. Sometimes the visual  meanings of the icons inside those buttons are vague.

So, I suggest to add some tooltips for the buttons.

**Comment by zalmoxisus** [comment]
Yes, we'll have tooltips for buttons in `3.0`.

**Comment by jkfinger** [comment]
This feature would be super valuable for new and existing users. I don't think I would have known this icon enabled Dispatch without my team. https://cl.ly/8a60f125f8e3/annotate

**Comment by sshadmand** [comment]
Yes please! And, until then, where are the docs for that list of buttons?
> This feature would be super valuable for new and existing users. I don't think I would have known this icon enabled Dispatch without my team. https://cl.ly/8a60f125f8e3/annotate

**Comment by sshadmand** [comment]
This was helpful for others looking: https://codeburst.io/redux-devtools-for-dummies-74566c597d7
